packages/botfuel-dialog/tests/adapters/messenger-adapter.test.js

```diff
@@ -87,3 +95,126 @@ describe('MessengerAdapter', () => {
     expect(extended).toHaveProperty('payload.value', 'message');
   });
 });
+
+describe('MessengerAdapter request handling', () => {
+  test('should handle a request from messenger', async () => {
+    // Data definitions
+
+    const profile = {
+      firstName: 'Sample',
+      lastName: 'User',
+    };
+    const userMessageUuid = '254a66a3-9282-4e4f-bd86-e18a376d5bc4';
+    const botMessageUuid = 'cbad2bc9-070c-4b37-b4be-190904fc506e';
+    const handlingTimestamp = 1526398451040;
```

Comment:
Maybe we can use Date.now() to generate a timestamp and for `responseTimestamp` add some delay to this timestamp?

Reply:
+1